stats/stats.go

```diff
@@ -205,6 +209,10 @@ func init() {
 // Record records a data point.
 func (h *Histogram) Record(v float64) {
 	h.Counter.Record(v)
+	h.appendRecordToHistogram(v)
+}
+
+func (h *Histogram) appendRecordToHistogram(v float64) {
```

Comment:
let's call this lowercase `record()` it's consistent with what it does - you could consider adding a count param to avoid a loop in the other one